qualtricssurvey/tests.py

```diff
@@ -30,11 +30,11 @@ def test_student_view(self):
         fragment = xblock.student_view()
 
         url_frag = (
-            'href="https://stanford.qualtrics.com/SE/?SID=Enter '
-            'your survey ID here.&amp;a='
+            'href="https://stanforduniversity.qualtrics.com/jfe/form/Enter '
```

Comment:
Nit per f2f: We may want to stick the base part of the URL here into a variable/constant.